examples/utilities.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+
+def ensure_no_files_overwritten(expected_contents):
+    # ensure we don't overwrite the user's files, if they run it on their machine
+    if os.path.exists(os.path.join(os.getcwd(), 'example.txt')):
+        with open("example.txt", "r") as f:
+            contents = f.read()
+        if contents != expected_contents:
+            print(
+                "I don't want to overwrite the example.txt file in my current directory! Please delete example.txt and try again!"
+            )
+            # exit with a non-zero error code
+            sys.exit(1)
```

Comment:
A neat trick with `sys.exit` is that you can pass it an error message string and it'll print it out and exit with a non-zero exit code. That lets you combine the `print` and `sys.exit` steps into a single call. It returns exit code 1 as well.

Reply:
Oh, I didn't know that, thanks!